### ISSUE TITLE
Expanding functionality of the client

### ISSUE THREAD

**Issue by akashagrahari** [Flusaka/dota-liquipedia-api-client#5]
Hello,

I recently came across this liquipedia client. I am working on an esports app which could use some of the existing functionality but would also require additional functionality - fetching information related to tournaments, etc.

Are you accepting contributions to expand the use-case of this client? If yes, I would love to contribute to it and use it in my application.

Looking forward to your response.

**Comment by Flusaka** [comment]
Absolutely! I haven't maintained this much or improved it at all but I have no problem with contributions!

If you wish you can make branches/pull requests from this and add the relevant changes you would like.

Sorry for the delayed reply, I only just noticed this 😅
